src/Catalyst.Core.Modules.Dfs/DfsService.cs

```diff
@@ -0,0 +1,663 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Abstractions.Dfs.BlockExchange;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Abstractions.Dfs.Migration;
+using Catalyst.Abstractions.Hashing;
+using Catalyst.Abstractions.Keystore;
+using Catalyst.Abstractions.Options;
+using Catalyst.Abstractions.Types;
+using Catalyst.Core.Lib.P2P;
+using Catalyst.Core.Modules.Dfs.BlockExchange;
+using Catalyst.Core.Modules.Dfs.Migration;
+using Catalyst.Core.Modules.Keystore;
+using Common.Logging;
+using Common.Logging.Serilog;
+using Lib.P2P;
+using Lib.P2P.Cryptography;
+using Lib.P2P.Discovery;
+using Lib.P2P.Protocols;
+using Lib.P2P.PubSub;
+using Lib.P2P.Routing;
+using Lib.P2P.SecureCommunication;
+using Makaretu.Dns;
+using MultiFormats;
+using Serilog;
+
+namespace Catalyst.Core.Modules.Dfs
+{
+    public class DfsService : IDfsService
+    {
+        static DfsService(){ /*LogManager.Adapter = new SerilogFactoryAdapter(Log.Logger); */ }
+
+        // (IPasswordManager passwordReader,
+        //             IFileSystem fileSystem,
+        //             ILogger logger,
+        //             string swarmKey = "07a8e9d0c43400927ab274b7fa443596b71e609bacae47bd958e5cd9f59d6ca3",
+        //             IEnumerable<MultiAddress> seedServers = null)
+        //         {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)